### ISSUE TITLE
[bug] babel-preset-typescript with babel-loader failed to parse private constructor

### ISSUE THREAD

**Issue by viktor-ku** [babel/babel#10644]
```
    "@babel/core": "^7.7.0",
    "@babel/preset-env": "^7.7.0",
    "@babel/preset-typescript": "^7.7.0",
    "babel-loader": "^8.0.6",
    "typescript": "^3.6.4",
    "webpack": "^4.40.2",
    "webpack-cli": "^3.3.9",
    "webpack-dev-middleware": "^3.7.0",
    "webpack-hot-middleware": "^2.25.0"
```

```ts
class Foo {
  private constructor() {}
}
```

in resulting js file I see 
```js
class Foo {
   private constructor() {}
}
```

**Comment by kryops** [comment]
We are experiencing similar problems since the 7.7.0 update:

```
ERROR in ../localization/src/locale.ts 18:10
Module parse failed: Unexpected token (18:10)
File was processed with these loaders:
 * ../../node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
|    * While the 'language' subtag is required, the other subtags are optional.
|    */
>   private constructor(language) {
```

**Comment by JLHwung** [comment]
Could you attach the babel config? I could not reproduce this issue with `@babel/preset-env` and `@babel/preset-typescript` enabled.

**Comment by kryops** [comment]
@JLHwung I can reproduce with just `@babel/preset-typescript`:

```
module.exports = function(api) {
  api.cache(true)

  return {
    presets: [
      '@babel/preset-typescript',
    ]
  }
}
```

Input:
```
class Foo {
  private constructor() {}

  // normal method for comparison
  private foo() {}
}
```

Output:
```
class Foo {
  private constructor() {} // normal method for comparison


  foo() {}

}
```

In combination with `@babel/preset-env` it can only be reproduced when targeting relatively modern browsers that support ES2015 classes.

I created a repo for it here: https://github.com/kryops/babel-issue-10644

Thank you for having a look at this! 😊

**Comment by JLHwung** [comment]
Fixed in 7.7.2.

**Comment by viktor-ku** [comment]
@JLHwung thanks a lot!
